app/src/interfaces/map/index.ts

```diff
@@ -1,15 +1,86 @@
-import { defineInterface } from '../define';
+import { defineInterface } from '@/interfaces/define';
+import { getBasemapSources } from '@/layouts/map/basemap';
 import InterfaceMap from './map.vue';
-import { sources, mapbox_sources } from '@/layouts/map/styles/sources';
 
 export default defineInterface({
 	id: 'map',
 	name: '$t:interfaces.map.map',
 	description: '$t:interfaces.map.description',
 	icon: 'map',
 	component: InterfaceMap,
-	types: ['json', 'csv'],
+	types: ['json', 'csv', 'string', 'text', 'binary', 'unknown'],
```

Comment:
How does this tie into https://github.com/directus/directus/discussions/5193? Should we add a single new type `geojson` and use that exclusively?

Wondering if the flexibility of all these types:

```
						{ value: 'geojson', text: 'GeoJSON' },
 						{ value: 'postgis', text: 'PostGIS' },
 						{ value: 'wkt', text: 'WKT' },
 						{ value: 'ewkt', text: 'EWKT' },
 						{ value: 'wkb', text: 'WKB' },
 						{ value: 'ewkb', text: 'EWKB' },
 						{ value: 'twkb', text: 'TWKB' },
 						{ value: 'csv', text: 'CSV' },
```

is worth the added complexity for people unfamiliar with what these are

Reply:
Yup, that's what I though too. If we go the route we discussed in #5193, then there would just be a `geometry` type (not `geojson`). The compatible types for the map layout and interface could then be `['json', 'geometry']`. People using WKT as a storage type probably are using spatial extensions on their database so they would fall in the `geometry` category.